code-challenges.test.js

```diff
@@ -19,15 +19,56 @@ const numbersArray1 = [6, 7, 8, 9, 10]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
+//Test: 
+//Function name: multBy3
+describe("multBy3",() => {
+  //Description of function
+  it("takes in an array of numbers and returns an array with all the numbers multipled by 3.",() => {
+    //Function: multBy3, Variable name: numbersArray1, expected output
+    expect (multBy3(numbersArray1)).toEqual([18, 21, 24, 27, 30]) 
+    //Function: multBy3, Variable name: numbersArray2, expected output
+    expect (multBy3(numbersArray2)).toEqual([72, 81, 90, 99, 108])
+  })
+  
+})
 
 // b) Create the function that makes the test pass.
-
-
+//Function: multBy3, Parameter: array
+const multBy3 = (array) => {
+    //Returns an array with the same amount of numbers, each multiplied by 3
+    return array.map(value => value * 3)
+  }
+  
+  console.log(multBy3(numbersArray1))
+  console.log(multBy3(numbersArray2))
+  
+  
+  
 
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+
+//Test: 
+//Function name: divisibleBy3
+describe("divisibleBy3",() => {
+//Description of function 
+  it("returns a string that says 'is divisible by three' or 'is not divisible by three'",() => {
+    //Function: divisibleBy3, Variable name: num1, expected output
+    expect(divisibleBy3(num1)).toEqual("15 is divisible by three")
+    //Function: divisibleBy3, Variable name: num2, expected output
+    expect(divisibleBy3(num2)).toEqual("0 is divisible by three")
+    //Function: divisibleBy3, Variable name: num3, expected output
+    expect(divisibleBy3(num3)).toEqual("-7 is not divisible by three")
+
+    })
+  
+})
```

Comment:
Test looks great! It is often best practice to place your test variables within the scope that they are being used. In this case, you are using the test variables in your it method which would be a great place to declare your test variables.